src/pudl/transform/eia860.py

```diff
@@ -356,6 +356,57 @@ def _core_eia860__generators(
     return gens_df
 
 
+@asset
+def _core_eia860__generators_energy_storage(
+    raw_eia860__generator_energy_storage_existing: pd.DataFrame,
+    raw_eia860__generator_energy_storage_retired: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform the energy storage specific generators table.
+
+    Notes for possible future cleaning:
+    * A lot of the values that are repeated in the generators table are dropped post
+    harvesting. It would be nice to compare the values before dropping them to make sure
+    these specific generator tables don't contain information that the big generator
+    tables does not.
+    """
+    storage_ex = raw_eia860__generator_energy_storage_existing
+    storage_re = raw_eia860__generator_energy_storage_retired
+
+    # every boolean column in the raw storage tables has a served_ or stored_ prefix
+    boolean_columns_to_fix = list(storage_ex.filter(regex="served_|stored_"))
```

Comment:
To ensure that it only matches at the beginning of the column name

```suggestion
    boolean_columns_to_fix = list(storage_ex.filter(regex="^(served_|stored_)"))
```

Reply:
oh great fix